### ISSUE TITLE
(Feature suggestion) Similar (But simplified) health pools for mobs.

### ISSUE THREAD

**Issue by ProjectAleph** [ichttt/FirstAid#163]
What if mobs had health pools similar to the player? You could fulfill your dream of headshotting a Skeleton to get revenge for every time they insta-killed you! It would absolutely add to the immersion this mod already brings.

One option that I came up with is having two separate health pools: one for the body and limbs, and one for the head. The body and limbs have a larger health pool, at the cost of having a larger hitbox. The head would be weaker and have a much smaller hitbox.

Not all mobs need this system. In my opinion, only humanoid mobs really need this system (one exception being the creeper, makes it more fun that way). 

One issue that implementing this could have is slower game speeds from having to do extra calculations.

**Comment by ichttt** [comment]
Yeah this is not something I have currently planned. Code-wise everything is build with the assumention of it only applying to players, which means it would probably be a lot of work as well. So probably never happening
